phoenix-queryserver/pom.xml

```diff
@@ -147,6 +147,14 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.phoenix</groupId>
+      <artifactId>phoenix-queryserver-loadbalancer</artifactId>
```

Comment:
I'm pretty sure I put this on the last PR (the one you closed). I think it's a bad idea to strongly tie PQS to the load balancer. You should make an interface in PQS which defines what it needs to know and then include the load balancer impl optionally (e.g. via ServiceLoader).